### DISCUSSION TITLE
move settings from osmand to osmand nigtly

### DISCUSSION THREAD

**Discussion by icegood** [osmandapp/OsmAnd#18540] [Q&A]
How do i move my favorite places/history/saved addresses etc from 
osmand to osmand nightly? 
Due to https://github.com/osmandapp/OsmAnd/issues/18539 osmand is not applicable for me

**Comment by DmitryAlexei** [comment]
You can use OsmAnd cloud sync (for more information see documentation https://osmand.net/docs/user/personal/osmand-cloud#osmand-start). You can sync settings and favorites for free or use advanced cloud backup with Pro subscription.
Furthermore, you can back up your data via menu > settings > swipe to the bottom of the screen > export to file
